lib/validators/options_processor/options/jira.js

```diff
@@ -0,0 +1,82 @@
+const JiraApi = require('jira-client')
+
+const ENABLED_NOT_FOUND_ERROR = `Failed to run the test because 'enabled' is not provided for 'jira' option. Please check README for more information about configuration`
+
+class Jira {
+  static async process (validatorContext, input, rule) {
+    let isMergeable
+    let regexObj
+
+    const filter = rule.jira
+    const enabled = filter['enabled']
```

Comment:
The enabled sub-option seem redundant to me. Maybe I'm missing something. Do you have a use-case where you have the `jira` option configured but you want to disable it? Wouldn't that be the same as just removing the `jira` option altogether?